megamek/src/megamek/client/bot/princess/BasicPathRanker.java

```diff
@@ -772,9 +779,9 @@ public double distanceToClosestEnemy(Entity me, Coords position, IGame game) {
         getOwner().methodBegin(BasicPathRanker.class, METHOD_NAME);
 
         try {
-            Entity closest = findClosestEnemy(me, position, game);
+            Targetable closest = findClosestEnemy(me, position, game);
             if (closest == null) {
-                return 0;
+                return -1;
```

Comment:
Indicates that no such thing exists, I should probably make a note of that in a comment.

Reply:
I concur that this should added, probably to the javadoc comment above